NuGet.config

```diff
@@ -23,96 +23,60 @@
     <!-- Used for BenchmarkDotNet prerelease packages -->
     <add key="benchmark-dotnet-prerelease" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/benchmark-dotnet-prerelease/nuget/v3/index.json" />
   </packageSources>
+  <!--
+    ****************************** AUTOGENERATED ***********************************
```

Comment:
@davidwengier my mappings seemed to be a lot less than yours originally. It's possible that the dependency changes made it simpler but also possible that I did a bad 😄

Reply:
Yeah, definitely very different. Will be interesting to see if the build works, particularly because I think the "dotnet-core-internal-tooling" source mapping was needed for arcade, so either arcade has changed and its not needed any more, or the build is gonna fail... or I did something wrong :)